pkgs/by-name/le/lerc/package.nix

```diff
@@ -23,14 +24,28 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-IHY9QtNYsxPz/ksxRMZGHleT+/bawfTYNVRSTAuYQ7Y=";
   };
 
-  patches = [
-    # https://github.com/Esri/lerc/pull/227
-    (fetchpatch {
-      name = "use-cmake-install-full-dir.patch";
-      url = "https://github.com/Esri/lerc/commit/5462ca7f7dfb38c65e16f5abfd96873af177a0f8.patch";
-      hash = "sha256-qaNR3QwLe0AB6vu1nXOh9KhlPdWM3DmgCJj4d0VdOUk=";
-    })
-  ];
+  # Required to get the freebsd-ports patch to apply.
+  # There seem to be inconsistent line endings in this project - just converting the patch doesn't work
+  prePatch = lib.optionalString stdenv.hostPlatform.isFreeBSD ''
+    ${buildPackages.dos2unix}/bin/dos2unix src/LercLib/fpl_EsriHuffman.cpp src/LercLib/fpl_Lerc2Ext.cpp
```

Comment:
optional:
```suggestion
    ${libbuildPackages.dos2unix} src/LercLib/fpl_EsriHuffman.cpp src/LercLib/fpl_Lerc2Ext.cpp
```

Reply:
Good idea, but:

> evaluation warning: getExe: Package "dos2unix-7.5.2" does not have the meta.mainProgram attribute. We'll assume that the main program has the same name for now, but this behavior is deprecated, because it leads to surprising errors when the assumption does not hold. If the package has a main program, please set `meta.mainProgram` in its definition to make this warning go away. Otherwise, if the package does not have a main program, or if you don't control its definition, use getExe' to specify the name to the program, such as lib.getExe' foo "bar".

Probably because it also contains unix2dos.